src/main/java/com/shippo/model/Address.java

```diff
@@ -33,6 +34,18 @@
     Object is_residential;
     Object metadata;
     Object messages;
+	ValidationResults validation_results;
+
+    public class ValidationResults {
+    	boolean is_valid;
+    	List<ValidationMessages> messages;
+	}
+
+	public class ValidationMessages {
```

Comment:
Can you add a test where you parse this information out from the response and verify there's a message with a `source`, `code`, and `text`?